src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,23 @@
 Release History
 ===============
 
+2.26.1
+++++++
+
+**ACR**
+
+* Hotfix: `az acr build\connected-registry\pack\run\scope-map`: Fix the compatibility bug caused by SDK upgrade
```

Comment:
Shouldn't we include the PR number in the release history?

Reply:
Updated

Reply:
I tried running that but ended up with this error: **'buildconnected-registrypackrunscope-map' is misspelled or not recognized by the system.**